subt/zmq-subt-x4.json

```diff
@@ -48,6 +48,25 @@
             "endpoint": "tcp://localhost:5555"
           }
       },
+      "drone": {
+          "driver": "subt.drone:Drone",
+          "in": ["desired_speed"],
+          "out": ["desired_speed_3d"],
+          "init": {
+          }
+      },
+      "localization": {
+        "driver": "subt.odoimuloc:Localization"
+      },
+      "vo": {
+        "driver": "subt.visual_odometry:Localization",
+        "init": {
+          "xyz": [0.23, 0, 0.25256005000000004],
+          "rpy": [0.0, 0.0, 0.0],
+          "principal-point": [320.5, 180.5],
+          "focal-length": 462.1
+        }
```

Comment:
Wasn't this part of some other PR already? I see the principal point is not correct here either. Can get current master merged into this branch? Thanks.